docs/installation/observability.rst

```diff
@@ -148,14 +149,113 @@ Open Forms produces application metrics (using Open Telemetry).
     enabled, but the code is in the Open Telemetry SDK instrumentation already and could
     possibly be opted-in to.
 
-**Application specific**
+Application specific
+--------------------
+
+Accounts
+^^^^^^^^
+
+``user_count``
+    Reports the number of users in the database. This is a global metric, you must take
+    care in de-duplicating results. Additional attributes are:
+
+    - ``scope`` - fixed, set to ``global`` to enable de-duplication.
```

Comment:
Could you please elaborate about de-duplication here? I don't quite understand it

Reply:
will do!